apps/dashboard/tests/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "commonjs"
```

Comment:
The `dal` and `testing` packages are CJS modules which can't be imported by the ESM modules. 
So this is the trick (together with tsconfig.json) to say "please allow me to import" in my test files :D